pkg/enqueue/Symfony/Consumption/LimitsExtensionsCommandTrait.php

```diff
@@ -58,6 +60,11 @@ protected function getLimitsExtensions(InputInterface $input, OutputInterface $o
             $extensions[] = new LimitConsumerMemoryExtension($memoryLimit);
         }
 
+        $niceness = (int) $input->getOption('niceness');
```

Comment:
we should not force int but check if the option is number and if not throw an exception with the message that explains it shuld be int.

Reply:
The check already is in constructor of extension. Do we really need additional one?